### ISSUE TITLE
Improve k8s-ci-bot language to be more clear

### ISSUE THREAD

**Issue by guineveresaenger** [kubernetes/test-infra#6469]
In k/test-infra's [contributing](https://github.com/kubernetes/test-infra/blob/master/CONTRIBUTING.md) instructions, the administrative requirements for having your pull request approved and lgtm'd is described fairly clearly.

The actual bot prompts should outline these requirements and remaining steps to take more clearly, with the contributor's level of authorization in mind.

Currently, if a non-owner submits a pull request, the bot response will say:

"[APPROVALNOTIFIER] This PR is NOT APPROVED"

followed by

"This pull-request has been approved by: <author>"

followed by instructions for code owners (not the reviewer).

As the non-owner author, I see "NOT APPROVED" followed by "approved by <me>" followed by "you can assign people" (feels like I'm intruding on someone's time) and "you can /approve as follows" (which is not really meant for me, the author).

**Comment by guineveresaenger** [comment]
According to [this document](https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md#the-code-review-process) the author of a pull request should also assign reviewers.

I have never done this. Nor has the bot prompted me to do this.

I have so far just poked people via slack or other channels that I knew would be interested. That is a barrier for new folks who are not connected (although it does somewhat encourage getting to know your SIG).

**Comment by cjwagner** [comment]
> According to this document the author of a pull request should also assign reviewers.
> 
> I have never done this. Nor has the bot prompted me to do this.

The bot should prompt you to `/assign` users in the approval notification (although this might not be the right place to mention this):
> Assign the PR to them by writing `/assign @cjwagner` in a comment when ready.


In some repos we have the [blunderbuss plugin](https://github.com/kubernetes/test-infra/tree/master/prow/plugins/blunderbuss) enabled. This plugin automatically requests reviews based on the `reviewers` section of OWNERS files.

**Comment by guineveresaenger** [comment]
Ah. Okay, the doc I linked is I think a bit wordy and we're taking it (and at least one similar doc) under review for the contributor guide.
